### ISSUE TITLE
Report title should be underneath actions toolbar

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-securityreport#34]
**SS 4.1.x-dev, CWP 2.0.x-dev, securityreport 2.0.0**

![image](https://user-images.githubusercontent.com/5170590/40593113-0c4aea66-627a-11e8-84ee-5891336fee3f.png)

As per https://github.com/silverstripe/silverstripe-reports/issues/100, the actions toolbar should be at the top of the report with content appearing underneath it rather than above it.

cc @clarkepaul

**Comment by ScopeyNZ** [comment]
FYI we did this in bringyourownideas/silverstripe-maintenance but as it's part of a GridField we chose to implement a `GridFieldHtmlFragment` GridField component:

https://github.com/bringyourownideas/silverstripe-maintenance/blob/master/src/Forms/GridFieldHtmlFragment.php

**Comment by robbieaverill** [comment]
Yeah this change needs to be made in the core silverstripe/reports module first IMO

**Comment by muskie9** [comment]
Would this require pulling over the class referenced, or would this be better suited in something like framework as a component for re-use? Happy to pull it into this module or reports if it doesn't make sense in framework.

**Comment by robbieaverill** [comment]
That example class was created because there wasn't something already available to do it in core. What do you think? I think we should move conversation to https://github.com/silverstripe/silverstripe-reports/issues/100 initially anyway, but we should aim to put as much of the customisation into framework -> reports -> here as possible, since this is a reusable pattern that we're trying to fix

**Comment by muskie9** [comment]
I think that makes sense. This is something that expects to utilize functionality that should be available already.
